src/query-builder/select-query-builder.ts

```diff
@@ -287,6 +295,18 @@ export class SelectQueryBuilder<DB, TB extends keyof DB, O>
     })
   }
 
+  havingNotExists(
+    arg: ExistsExpression<DB, TB>
+  ): SelectQueryBuilder<DB, TB, O> {
+    return new SelectQueryBuilder({
+      ...this.#props,
+      queryNode: SelectQueryNode.cloneWithHaving(
+        this.#props.queryNode,
+        parseNotExists(arg)
+      ),
+    })
+  }
+
```

Comment:
Should this be deprecated as well?

Reply:
It is. It gets the @deprecated doc from the interface. We used to have a bug where the function names was wrong.

Reply:
I see. OK!